controllers/provisioning_controller.go

```diff
@@ -140,6 +142,21 @@ func (r *ProvisioningReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 		// Temporarily not requeuing request
 		return ctrl.Result{}, nil
 	}
+
+	// Read container images from Config Map
+	var containerImages Images
+	if err := GetContainerImages(&containerImages, ContainerImagesFile); err != nil {
+		// Images config map is not valid
+		// Provisioning configuration is not valid.
+		// Requeue request.
+		r.Log.Error(err, "invalid contents in images Config Map")
+		co_err := r.updateCOStatusDegraded("invalid contents in images Config Map", err.Error())
```

Comment:
Do we know it is a ConfigMap at this point? Can we include its name in the error?